Tests/XcodeGenKitTests/SpecLoadingTests.swift

```diff
@@ -590,6 +590,28 @@ class SpecLoadingTests: XCTestCase {
                 try expect(target.sources) == ["nestedTemplateSource2", "nestedTemplateSource1", "templateSource", "targetSource"] // merges array in order
                 try expect(target.configFiles["debug"]) == "Configs/Framework/debug.xcconfig" // replaces $target_name
             }
+            
+            $0.it("parses cross platform target templates") {
+  
+                let project = try getProjectSpec([
+                    "targets": [
+                        "Framework": [
+                            "type": "framework",
+                            "templates": ["temp"],
+                        ]
+                    ],
+                    "targetTemplates": [
+                        "temp": [
+                            "platform": ["iOS", "tvOS"],
+                        ]
+                    ],
+                    ])
+                
+                let iOSTarget = project.targets.first { $0.platform == .iOS }
+                let tvOSTarget = project.targets.first { $0.platform == .tvOS }
+                try expect(iOSTarget?.type) == .framework
+                try expect(tvOSTarget?.type) == .framework
```

Comment:
The solution should be more generic, something like that:
```
let platforms: [Platform] = [.iOS, .tvOS]
platforms.forEach { platform in
    let target = project.targets.first { $0.platform == platform }
    try expect(target?.type) == .framework
}
```
Or maybe you should use `Platform.all` here.